ig/input/fsh/VaccinationScheme.fsh

```diff
@@ -6,9 +6,56 @@ Description: """
   VaccinationScheme models a process consisting of immunizations through wich a patient can reach immunity against a target
   disease.
 """
-* extension contains VPVaccinationSchemeExtension named vaccinationSchemeExtension 1..1
+* extension contains VPVaccinationSchemeExtension named vaccinationSchemeExtension 1..1 MS
 
 Extension: VPVaccinationSchemeExtension
 Id: vp-vaccination-scheme-extension
 Title: "VaccinationSchemeExtension"
 Description: ""
+* extension contains
+  name 1..1 MS and
+  type 1..1 MS and
+  isPreferred 1..1 MS and
+  ageStart 0..1 MS and
+  ageEnd 0..1 MS
+
+* extension[name]
+  * value[x] 1..1 MS
+  * value[x] only string
+
+* extension[type]
+  * value[x] 1..1 MS
+  * value[x] only string
+  * value[x] from VPVaccinationSchemeTypeValueSet
+
+* extension[isPreferred]
+  * value[x] 1..1 MS
+  * value[x] only boolean
+
+* extension[ageStart]
+  * value[x] 1..1 MS
+  * value[x] only Quantity
+  * value[x] obeys age-1
+  * value[x].code from http://hl7.org/fhir/ValueSet/all-time-units
```

Comment:
I think we should set value, system, code (and maybe even unit) of value[x] to `1..1 MS`

The same applies to ageEnd.

Reply:
Addressed at f6edd73d